src/Compilers/CSharp/Portable/Binder/TypeofBinder.cs

```diff
@@ -23,17 +23,14 @@ namespace Microsoft.CodeAnalysis.CSharp
     internal sealed class TypeofBinder : Binder
     {
         private readonly Dictionary<GenericNameSyntax, bool> _allowedMap;
-        private readonly bool _isTypeExpressionOpen;
```

Comment:
Unused field. (only read in unused property).

Reply:
discovered while debugging through compiler to determine how unbound generics are bound.